lib/analytics/widgets/analytics_cart_listener.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:async';
+
+import 'package:flex_storefront/analytics/apis/analytics_api.dart';
+import 'package:flex_storefront/analytics/models/analytics_events.dart';
+import 'package:flex_storefront/cart/cart_repository.dart';
+import 'package:flex_storefront/cart/models/cart_message.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+
+class CartMessageListener extends StatefulWidget {
```

Comment:
Can we please align the filename with the class name?

Reply:
Oh that was a mistake, yes totally 👍

Reply:
I actually did the opposite: renaming the class to `AnalyticsCartListener` since it feels more than an analytics thing than a cart thing.